commands/errors.go

```diff
@@ -58,6 +58,7 @@ var (
 	ErrMustBeEmpty                                     = errors.New("must be empty")
 	ErrMustBeGTEClampLowerBound                        = errors.New("must be greater than or equal to clamp lower bound")
 	ErrOneTimeTriggerAllowedMax                        = errors.New("maximum one time trigger allowed")
+	ErrMustBeBetween01                                 = errors.New("must be in range (0,1]")
```

Comment:
Since this is user facing I think we should have something a bit more user friendly, like the other ones. I agree we are missing the boundary on the others though, so what about:
```suggestion
	ErrMustBeBetween01                                 = errors.New("must be between 0 (excluded) and 1 (included)")
```

For the story, in France, we don't use parenthesis to defined ranges, when learning at school. We only use square brackets. That would be `]0,1]`. So, with words, we can avoid that cultural difference.

Reply:
I don't know French, but in maths the meaning of (0,1] is 0<x<=1. I've no problem with having it more verbose as suggested.

Reply:
I am happy with either notation `]a,b]` or `(a,b]` but in user facing stuff I agree we should be explicit.

Reply:
@ValentinTrinque - arguably the variable name is dangerous because someone may use `ErrMustBeBetween01` for `(0,1)` or something else that "between 0 and 1".